src/integration-test/tests/citizen/directionsQuestionnaire/pages/support-required.ts

```diff
@@ -0,0 +1,42 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  disabledAccess: {
+    input: 'input[id="disabledAccessSelectedtrue"]'
+  },
+  hearingLoop: {
+    input: 'input[id="hearingLoopSelectedtrue"]'
+  },
+  signLanguageInterpreter: {
+    input: 'input[id="signLanguageSelectedtrue"]',
+    details: 'input[id="signLanguageInterpreted"]'
+  },
+  languageInterpreter: {
+    input: 'input[id="languageSelectedtrue"]',
+    details: 'input[id="languageInterpreted"]'
+  },
+  otherSupport: {
+    input: 'input[id="otherSupportSelectedtrue"]',
+    details: 'textarea[id="otherSupport"]'
+  }
+}
+
+const buttons = {
+  submit: 'input[type=submit]'
+}
+
+export class SupportRequiredPage {
+  selectAll (someText: string): void {
```

Comment:
is there any need to pass in someText as a variable? just hard code in something

Reply:
done